components/gems/Gemfile

```diff
@@ -21,7 +21,7 @@ group(:omnibus_package, :development, :test) do
   # Require a minimum version that is packaged in the Ruby install, so we do
   # not install old unecessary versions. When we bump Ruby we need to look
   # at these pins and adjust them.
-  gem "minitest", "= 5.13.0"
+  gem "minitest", "= 5.14.2"
 
   gem "rake", "= 13.0.1" # prevent double rake as of 12.2020
```

Comment:
I think we can remove this entirely now that we've cleaned up some of our omnibus mess